easistrain/EDD/strainStressd0cstEDD.py

```diff
@@ -0,0 +1,224 @@
+
+import numpy as np
+import h5py
+import scipy.optimize
+
+# fileRead = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_globstrain_TiC.h5'
+# fileSave = '/home/esrf/slim/easistrain/easistrain/EDD/Results_ihme10_strainStressTensor_TiC.h5'
+# numberOfPeaks = 5
+# XEC = [S1(hkl1), dS2(hkl1), S1(hkl2), dS2(hkl2), S1(hkl3), dS2(hkl3), S1(hkl4), dS2(hkl4), S1(hkl5), dS2(hkl5)]
+
+
+
+
+def deforDirMeas(angles, e11, e22, e33, e23, e13, e12):
+    phi = np.radians(angles[:, 0])
+    chi = np.radians(angles[:, 1])
+    omega = np.radians(angles[:, 2])
+    delta = np.radians(angles[:, 3])
+    theta = np.radians(0.5*angles[:, 4])
+    q1 = (
```

Comment:
Is this the same formula as in `diffVector` ?

If so, you can reuse the function: `from .preStraind0cstEDD import diffVector`